src/local_sensitivity/concrete_solve.jl

```diff
@@ -154,10 +155,10 @@ function DiffEqBase._concrete_solve_adjoint(prob,alg,
       end
     end
 
-    if haskey(kwargs, :callback_adj)
+    if haskey(kwargs_adj, :callback_adj)
       cb2 = CallbackSet(cb,kwargs[:callback_adj])
     else
-      cb2 = nothing
+      cb2 = cb

```

Comment:
This enabled the passing tests in the previous test runs.. The reverse callback was actually not triggered.